dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HadoopUtils.java

```diff
@@ -420,9 +419,7 @@ public ExecutionStatus getApplicationStatus(String applicationId) throws Excepti
 
         String result = Constants.FAILED;
         String applicationUrl = getApplicationUrl(applicationId);
-        if (logger.isDebugEnabled()) {
-            logger.debug("generate yarn application url, applicationUrl={}", applicationUrl);
-        }
+        logger.info("applicationUrl={}", applicationUrl);
```

Comment:
The outer loop calls this method, and if the log level is changed to INFO here, This information will always be printed(the YARN task runtime)

Reply:
Sorry, I did not modify the code, it may be caused by a git conflict